dev/README_RELEASE_PROVIDER_PACKAGES.md

```diff
@@ -392,8 +393,7 @@ breeze release-management publish-docs \
     --package-filter 'apache-airflow-providers-*' \
     --override-versioned
 
-cd "${AIRFLOW_SITE_DIRECTORY}"
```

Comment:
If you remove this cd then please remove also the `cd ..` in line 397

Reply:
Noted. Will keep in mind while updating the PE

Reply:
BTW. I think the best thing is to just ... execute the processs... this way is the only way to check if it works @amoghrajesh - and it does not have to be complete. Like pushing the code does not neede to happen, and up to pushing to airflow repo, all steps can be done by you as well.

Reply:
Thanks for that info, @potiuk
I will try it out now and in the future PRs too!